src/buildSummary.ts

```diff
@@ -0,0 +1,59 @@
+// Copyright 2024 The MathWorks, Inc.
+import * as core from "@actions/core";
+import { join } from 'path';
+import { readFileSync } from 'fs';
+
+export interface Task {
+  name: string;
+  description: string;
+  failed: boolean;
+  skipped: boolean;
+  duration: string;
+}
+
+
+export function getBuildSummaryTable(tasks: Task[]): string[][] {
+  const header: string[] = ['MATLAB Build Task', 'Status', 'Description', 'Duration (HH:MM:SS)'];
+  let taskSummaryTableRows: string[][] = [header];
+
+  tasks.forEach((task, index) => {
```

Comment:
The indentation of this file looks different than the indentation of other files in the project. This is using 2 spaces. The other files appear to be using 4 spaces.

Reply:
Done.